### ISSUE TITLE
Ingress error on v1.15.0: Required value: pathType must be specified

### ISSUE THREAD

**Issue by pablorsk** [loft-sh/devspace#1637]
After upgrade devspace from `5.14.4` to `1.15.0`, deployment ingress return an error:

```
[fatal]  error deploying main: Unable to deploy helm chart: error during 'helm upgrade main 
/home/mysuer/.devspace/component-chart/component-chart-0.8.1.tgz
 --namespace my-namespace --values /tmp/894348912 --install --kube-context do-sfo2-rs-cluster': 
Error: UPGRADE FAILED: cannot patch "main" with kind Ingress: Ingress.extensions "main" is invalid: 
[spec.rules[0].http.paths[0].pathType: Required value: pathType must be specified, 
spec.rules[1].http.paths[0].pathType: Required value: pathType must be specified, 
spec.rules[2].http.paths[0].pathType: Required value: pathType must be specified, 
spec.rules[3].http.paths[0].pathType: Required value: pathType must be specified]
 => exit status 1
Run `devspace purge -d main` and `devspace deploy` to recreate the chart
```

I downgrade to `5.14.4`, and all works fine.

**Local Environment:**  
- DevSpace Version: 5.15.0
- Operating System: linux
- Deployment method: helm

**Kubernetes Cluster:**  
- Cloud Provider: other
- Kubernetes Version: v1.20.2

/kind bug

**Comment by cheslijones** [comment]
I think I am running into a similar issue:

```
deployments:
- name: client
  helm:
    componentChart: true
    values:
      containers:
      - image: app/client
      service:
        ports:
          - port: 3000
      ingress:
        rules: 
        - host: localhost
          path: /
          #  pathType: Prefix
```
This results in:
```
Error: Ingress.extensions "client" is invalid: spec.rules[0].http.paths[0].pathType: Required value: pathType must be specified
 => exit status 1
Run `devspace purge -d client` and `devspace deploy` to recreate the chart
```
But uncommenting `pathType: Prefix` such that:
```
deployments:
- name: client
  helm:
    componentChart: true
    values:
      containers:
      - image: app/client
      service:
        ports:
          - port: 3000
      ingress:
        rules: 
        - host: localhost
          path: /
          pathType: Prefix
```
Results in:
```
[fatal]  deployments[1].helm.componentChart: component values are incorrect: yaml: unmarshal errors:
  line 7: field pathType not found in type latest.IngressRuleConfig
```

**Comment by FabianKramm** [comment]
@cheslijones this should work with the new version v5.16.0-alpha.0 now, we'll release a stable version really soon

**Comment by FabianKramm** [comment]
Should be fixed with v5.16.0
